scrapy/core/downloader/handlers/http11.py

```diff
@@ -151,23 +157,22 @@ def connect(self, protocolFactory):
         return self._tunnelReadyDeferred
 
 
-def tunnel_request_data(host, port, proxy_auth_header=None):
+def tunnel_request_data(host, port, proxy_auth_header=None, host_header=True):
```

Comment:
Noticed that while reading coverage report - host_header was always True in tests. Let's drop it :) +1 to merge the PR after that.